src/components/Task.js

```diff
@@ -3,7 +3,7 @@ import { FaTimes } from 'react-icons/fa'
 const Task = ({ task, onDelete, onToggle }) => {
   return (
     <div
-      className={`task ${task.reminder ? 'reminder' : ''}`}
+      className={`task ${!task.reminder && 'reminder'}`}
```

Comment:
```suggestion
      className={`task ${task.reminder && 'reminder'}`}
```
Isn't it add `reminder` class only when `task.reminder` returns true?